misc/acrn-config/library/launch_cfg_lib.py

```diff
@@ -578,12 +578,13 @@ def get_slot(bdf_list, dev):
             slot_fun = str(bus) + ":" +  str(slot) + ":" + str(fun)
             if bus != 0:
```

Comment:
what does bus!=0 mean here? why need to get a new virtual slot when bus is not 0?

Reply:
'bus' is PCI bus number, if user select PCI device 03:00.0 and 00:03.0 for passthru in UI, this would cause slot conflict in launch script. So get a new virtual slot if bus number is not 0.